writers/cassandra/messages_test.go

```diff
@@ -0,0 +1,29 @@
+package cassandra_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/mainflux/mainflux"
+	"github.com/mainflux/mainflux/writers/cassandra"
+	"github.com/stretchr/testify/assert"
+)
+
+const keyspace = "mainflux"
+
+var (
+	msg  = mainflux.Message{}
+	addr = "localhost"
+)
+
+func TestSave(t *testing.T) {
+	session, err := cassandra.Connect([]string{addr}, keyspace)
+	if err != nil {
```

Comment:
Assert error is  `nil`.

Reply:
Assert is used only if you want your test to continue running. There is no point in that here. Here I need fatal.

Reply:
In that case, since we are using `testify`, consider using [https://github.com/stretchr/testify/tree/master/require](https://github.com/stretchr/testify/tree/master/require).